sdk/identity/azure-identity/src/main/java/com/azure/identity/package-info.java

```diff
@@ -2,7 +2,277 @@
+ * recommended to use that specific credential to authenticate. Let's take a look at the individual
+ * authentication scenarios and their respective credential use below.</p>
+ *
+ * <hr/>
+ *
+ * <h2> Authenticating on Azure Hosted Platforms via Managed Identity</h2>
+ *
+ * <p>The {@link com.azure.identity.ManagedIdentityCredential} authenticates the configured managed identity
+ * (system or user assigned) of an Azure resource. So, if the application is running inside an Azure resource that
+ * supports Managed Identity through IDENTITY/MSI, IMDS endpoints, or both, then the
+ * {@link com.azure.identity.ManagedIdentityCredential} will get your application authenticated, and offers a great
+ * secretless authentication experience. For more information refer to the
+ * <a href="https://aka.ms/azsdk/java/identity/managedidentity/docs">conceptual knowledge and configuration
+ * details</a>.</p>
+ *
+ * <p><strong>Sample: Construct a Managed Identity Credential</strong></p>
+ *
+ * <!-- src_embed com.azure.identity.credential.managedidentitycredential.construct -->
+ * <pre>
+ * TokenCredential managedIdentityCredential = new ManagedIdentityCredentialBuilder&#40;&#41;
+ *     .build&#40;&#41;;
+ * </pre>
+ * <!-- end com.azure.identity.credential.managedidentitycredential.construct -->
+ *
+ * <p>For other credentials that work well in Azure Hosted platforms, refer to the table below.</p>
+ *
+ * <br/>
+ *
+ * <table style="border: 2px; width: 50%;">
+ *   <caption><strong>Authenticate Azure-hosted applications</strong></caption>
+ *   <thead>
+ *     <tr>
+ *       <th>Credential class</th>
+ *       <th>Usage</th>
+ *     </tr>
+ *   </thead>
+ *   <tbody>
+ *     <tr>
+ *       <td>{@link com.azure.identity.DefaultAzureCredential}</td>
+ *       <td>Provides a simplified authentication experience to quickly start developing applications run in Azure.
+ *       </td>
+ *     </tr>
+ *     <tr>
+ *       <td>{@link com.azure.identity.ChainedTokenCredential}</td>
+ *       <td>Allows users to define custom authentication flows composing multiple credentials.</td>
+ *     </tr>
+ *     <tr>
+ *       <td>{@link com.azure.identity.EnvironmentCredential}</td>
+ *       <td>Authenticates a service principal or user via credential information specified in environment variables.
+ *       </td>
+ *       <td></td>
+ *     </tr>
+ *     <tr>
+ *       <td>{@link com.azure.identity.ManagedIdentityCredential}</td>
+ *       <td>Authenticates the managed identity of an Azure resource.</td>
+ *     </tr>
+ *   </tbody>
+ * </table>
+ *
+ * <br/>
+ *
+ * <p>Refer to the JavaDoc for each of these classes for more details about when these credential types should be used.
+ * </p>
+ *
+ * <hr/>
+ *
+ * <h2>Authenticate with Service Principals</h2>
+ *
+ * <p>The Azure Active Directory (Azure AD) allows users to register service principals which can be used as an
```

Comment:
Is it "The Azure Active Directory allows..." or "Azure Active Directory allows..."? I would have assumed it was the latter.

Reply:
Brief intro paragraph on what a "service principal" is.